src/components/RulesTextBox/RulesTextBox.js

```diff
@@ -0,0 +1,16 @@
+import { html } from '../../shared';
+import styles from './RulesTextBox.module.css';
+
+/**
+ * @param {string} header
+ * @param {string} text
+ * @returns {HTMLElement}
+ */
+function RulesBox(header, text) {
```

Comment:
Technicznie rzecz biorąc to nie błąd, ale trochę mylące jest, że wszędzie mamy nazwę `RulesTextBox`, a tylko w jednym miejscu `RulesBox`.

Reply:
zmienione. Dzięki za podpowiedź